mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule Jetstream.MixProject do
   use Mix.Project
 
-  @version "0.0.2"
+  @version "0.0.3"
```

Comment:
I think bumping version number is out of scope of this PR 😃

Reply:
https://github.com/mmmries/jetstream/pull/64/commits/2ce7e538ec2cf5668289824b375193caa74ca090